src/elf.rs

```diff
@@ -230,6 +230,11 @@ impl SBPFVersion {
         self != &SBPFVersion::V1
     }
 
+    /// Use src reg instead of imm in callx
+    pub fn fix_callx_encoding(&self) -> bool {
```

Comment:
nit: call it callx_uses_src_reg or something? "fix" is generic: what was broken?
how was it fixed?